### ISSUE TITLE
WINDOW 7.8.14 - Reverse calculation takes single-panel glass parameters

### ISSUE THREAD

**Issue by vukhang-93** [LBNL-ETA/Windows-CalcEngine#166]
Currently when using WINDOW 7.8.14, we need to enter the parameters of 2 single glass panels to get the parameters of an insulated glass panel. I want to ask, is there any way to get the parameters of the remaining single glass panel when we know the parameters of the insulated glass panel and the parameters of a single glass panel?

**Comment by vidanovic** [comment]
No. It is not possible with this engine nor WINDOW.

**Comment by vukhang-93** [comment]
@vidanovic yes, thank you for your response. I'm happy to use Window. It helps me a lot with my work. Please develop it further and update new information related to it. I will follow up regularly.
